crawler.py

```diff
@@ -244,6 +244,11 @@ def start_driver(self):
         self.driver.set_page_load_timeout(self.timeout)
         self.driver.set_script_timeout(self.timeout)
 
+        # log the browser type and version
+        self.logger.info((
+        'Running scan with %s (v. %s)'),
+        self.driver.capabilities["browserName"], self.driver.capabilities["browserVersion"])
```

Comment:
We already have a line to log the browser type in the summary text. Let's log the browser version there as well.